Firmware/mmu2.cpp

```diff
@@ -113,9 +113,7 @@ MMU2::MMU2()
     , unloadFilamentStarted(false)
     , loadingToNozzle(false)
     , inAutoRetry(false)
-{
-    ResetRetryAttempts();
-}
+{}
```

Comment:
If you remove ResetRetryAttempts, you should at least initialize the variable.

Reply:
I moved it to Start(). Isn’t that enough or is it initialized too late?

Reply:
It is a good practise to initialize all members of a struct/class in the constructor. In this case Start() does the usual reinit stuff and it would work unless someone decides to introduce some PreStartSomething which may rely on the retryAttempts being initialized.